components/Header/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="header.css"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Document</title>
+</head>
+<body>
+    <nav class='navigation-bar' >
+        <div class='list-parent-left' id='symbol'>
+            <a class='li-link'href='../../index.html'> <h3>Chikku<span style='color:rgb(10, 177, 10)'>Ui</span></h3> </a>  
+        </div>
+        <div class='list-parent-right' id='modal'>
+            <a  href="javascript:void(0);" class="li-icon" onclick="modelFunction()">
+                <i class="fa fa-bars"></i>
+              </a>
+              <a class='li-link' href='../../index.html'> Home</a>
+              <a class='li-link' href='components/alert/index.html'>Documentation</a>
+              <a class='li-link' target='_blank' href='https://github.com/vineeth-t/Main/tree/Devlopment'>Github</a>
+              <!-- <a class='li-link'href='#'>V0.1</a> -->
+        </div>
+        </nav>
+    <script type="text/javascript" src="header.js"></script>
+</body>
+</html>
+<!-- <div class='list-parent-left'>
```

Comment:
can remove unused code

Reply:
This is alternate to the above div .I have used lists in the commented section where as above code uses div's